dataikuapi/dss/dataset.py

```diff
@@ -859,3 +870,78 @@ def already_exists(self):
             return True
         except Exception as e:
             return False
+
+
+class DSSDatasetInfo(object):
+    """
+    Info class for a DSS dataset (Read-Only).
+    Do not instantiate this class directly, use :meth:`DSSDataset.get_info`
+    """
+
+    def __init__(self, dataset, info):
+        """
```

Comment:
Nitpick: we don't normally document constructors that we ask customers not to call

Reply:
removed 👍🏻